### PR TITLE
update: added missed step about adding dependency to the web socket s…

### PR DESCRIPTION
**Pull request by KaterinaPetrova**
…ample of using platform-specific apis

Context: https://youtrack.jetbrains.com/issue/KT-43121#focus=Comments-27-4630443.0-0

### CODE REVIEW

File: topics/connect-to-platform-specific-apis.md [hunk 1]
```diff
@@ -135,6 +135,20 @@ internal actual class PlatformSocket actual constructor(url: String) {
 }
 ```
 
+Android implementation uses [OkHttp](https://square.github.io/okhttp/) third-party library, so you need to declare the corresponding dependency in the shared module `build.gradle`:
```
Comment (katia-energizer) [inline topics/connect-to-platform-specific-apis.md:138]:
```suggestion
Android implementation uses the third-party library [OkHttp](https://square.github.io/okhttp/). Add the corresponding dependency to `build.gradle(.kts)` in the shared module:
```

File: topics/connect-to-platform-specific-apis.md [hunk 2]
```diff
@@ -135,6 +135,20 @@ internal actual class PlatformSocket actual constructor(url: String) {
...
+
+```kotlin
```
Comment (katia-energizer) [inline topics/connect-to-platform-specific-apis.md:140]: Please provide a Groovy sample as well.

File: topics/connect-to-platform-specific-apis.md [hunk 3]
```diff
@@ -135,6 +135,20 @@ internal actual class PlatformSocket actual constructor(url: String) {
...
+sourceSets {
+    val androidMain by getting {
+        dependencies {
+            implementation("com.squareup.okhttp3:okhttp:4.9.0")
```
Comment (katia-energizer) [inline topics/connect-to-platform-specific-apis.md:144]: Is it OK to use specific version here? Will it work in the future?

File: topics/connect-to-platform-specific-apis.md [hunk 4]
```diff
@@ -135,6 +135,20 @@ internal actual class PlatformSocket actual constructor(url: String) {
...
+        }
+    }
+}
+```
+
+iOS implementation uses `NSURLSession` from the standard Apple SDK, so you don't need to add any other dependencies.
```
Comment (katia-energizer) [inline topics/connect-to-platform-specific-apis.md:150]:
```suggestion
iOS implementation uses `NSURLSession` from the standard Apple SDK and doesn't require additional dependencies. 
```

**Review by katia-energizer — CHANGES_REQUESTED** [review]
Thanks for updating! Please take a look and suggestions and comments.

File: topics/connect-to-platform-specific-apis.md (hunk 3 above)
Comment (KaterinaPetrova) [reply topics/connect-to-platform-specific-apis.md]: Fixed

**Review by katia-energizer — APPROVED** [review]
Everything is OK.

**Pull request merged**